### ISSUE TITLE
Does it work with PhpStorm9? NullPointerException

### ISSUE THREAD

**Issue by MantasVaitkunas** [Haehnchen/idea-php-oxid-plugin#1]
I'm getting errors when trying to call public method. Let's say I create object $oOrder = oxNew('oxOrder'); and I try to call method $oOrder->getOrderUser(), so auto completion for method does not work. I get error:

```
null
java.lang.NullPointerException
    at de.espend.idea.oxid.utils.ModuleUtil.getExtendsList(ModuleUtil.java:105)
    at de.espend.idea.oxid.utils.OxidUtil.getPseudoClassOverwrites(OxidUtil.java:55)
    at de.espend.idea.oxid.utils.OxidUtil$1.run(OxidUtil.java:143)
    at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:872)
    at de.espend.idea.oxid.utils.OxidUtil.buildClassMetadataFile(OxidUtil.java:139)
    at de.espend.idea.oxid.OxidProjectComponent$1$1.run(OxidProjectComponent.java:53)
    at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:563)
    at com.intellij.openapi.progress.impl.CoreProgressManager$2.run(CoreProgressManager.java:152)
    at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:452)
    at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:402)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
    at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:137)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl$1.run(ProgressManagerImpl.java:126)
    at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:400)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
    at java.util.concurrent.FutureTask.run(FutureTask.java:262)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
    at java.lang.Thread.run(Thread.java:745)
    at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:56)
```

**Comment by Haehnchen** [comment]
it works fine in PhpStorm9. there is somewhere an invalid array condition in `extend` metadata of your project. i did not catch this, will fix it.

**Comment by Haehnchen** [comment]
@MantasVaitkunas fixed in 0.2

**Comment by MantasVaitkunas** [comment]
Seems like everything works well now :thumbsup:
